express-post-json/index.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const app = express();
+let nextID = 1;
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  const gradesArray = [];
+  for (id in grades) {
+    gradesArray.push(grades[id]);
+  }
+  res.json(gradesArray);
+});
+
+app.listen(3000, () => {
+  console.log('Listening to Port 3000!');
+});
+
+const parsedJSON = express.json();
+
+app.use(parsedJSON);
+
+app.post('/api/grades', (req, res) => {
+  if (!req.body) {
+    res.sendStatus(404);
+  } else {
```

Comment:
`req.body` will never be falsy if you have successfully added the JSON parsing middleware. It's guaranteed to be at least an empty object (which is truthy). You can remove this conditional statement.